pyci/utility.py

```diff
@@ -147,3 +147,83 @@ def spinize_rdms(d1, d2):
         abba -= np.swapaxes(d2[2], 2, 3)  # -abab
         baab -= np.swapaxes(d2[2], 0, 1)  # -abab
     return rdm1, rdm2
+
+
+def odometer_one_spin(wfn, nodes, t, qmax):
```

Comment:
Because odometer works right now with the general cost function (e.g energies, number of nodes, etc. ) it's better to change name of parameter `nodes` to `cost`